ShipperOptions.json

```diff
@@ -2604,6 +2604,18 @@
         "value": "USPS_PTP_PSBN",
```

Comment:
i think we could delete this one too, pretty sure USPS Parcel Select is now in the UGA

Reply:
we need to leave it because we use this file for displaying shipping methods from old orders.

> You can check-out any time you like
> But you can never leave